functions/utils.ts

```diff
@@ -37,6 +37,14 @@ const importProject = async (url: string): Promise<ProjectInfo> => {
 };
 
 export const getProjectInfo = async (url: URL): Promise<ProjectInfo> => {
+  let title = url.searchParams.get('title')
```

Comment:
please use `const` instead of `let` to [make eslint happy](https://github.com/live-codes/livecodes/actions/runs/14988103780/job/42110921403?pr=819) :)